GVFS/GVFS.Common/LocalGVFSConfig.cs

```diff
@@ -76,15 +76,20 @@ public LocalGVFSConfig()
                 const string ErrorFormat = "Error setting config value {0}: {1}. Config file {2}. {3}";
                 if (tracer != null)
                 {
-                    tracer.RelatedError(ErrorFormat, key, value, this.configFile, exception.ToString());
+                    tracer.RelatedError(ErrorFormat, name, value, this.configFile, exception.ToString());
                 }
 
-                error = string.Format(ErrorFormat, key, value, this.configFile, exception.Message);
+                error = string.Format(ErrorFormat, name, value, this.configFile, exception.Message);
                 value = null;
                 return false;
             }
         }
 
+        private string KeyFromConfigName(string configName)
+        {
+            return configName.ToUpperInvariant();

```

Comment:
I don't think we should change what the user is telling us to store, if we can avoid it. What are your thoughts on allowing for case-insensitive lookups in the dictionary?

Reply:
Have updated the dictionary to accept a case insensitive comparer. Removed all ToUpper() calls. Is this similar to what you had in mind?